src/components/molecules/PluginEditor/index.tsx

```diff
@@ -120,139 +58,141 @@ const PluginEditor: React.FC = () => {
+                      onClick={() => setInfoboxSize("medium")}>
+                      Medium
+                    </Button>
+                    <Button
+                      selected={infoboxSize === "large"}
+                      onClick={() => setInfoboxSize("large")}>
+                      Large
+                    </Button>
                   </div>
-                ))}
-              </div>
-              <div style={{ display: "flex", flexDirection: "column", margin: "4px 0" }}>
-                <Button selected={showInfobox} onClick={handleInfoboxToggle}>
-                  Toggle Infobox
-                </Button>
+                </div>
                 <div
                   style={{
                     display: "flex",
-                    flexDirection: "column",
-                    maxHeight: `${showInfobox ? "100px" : "0"}`,
-                    paddingBottom: `${showAlignSystem ? "2px" : "0"}`,
-                    overflow: "hidden",
-                    transition: "all 1s",
-                    borderBottom: `${showInfobox ? "1px solid white" : "none"}`,
+                    justifyContent: "space-between",
                   }}>
-                  <Button
-                    selected={infoboxSize === "small"}
-                    onClick={() => setInfoboxSize("small")}>
-                    Small
-                  </Button>
-                  <Button
-                    selected={infoboxSize === "medium"}
-                    onClick={() => setInfoboxSize("medium")}>
-                    Medium
-                  </Button>
-                  <Button
-                    selected={infoboxSize === "large"}
-                    onClick={() => setInfoboxSize("large")}>
-                    Large
-                  </Button>
+                  <p>Extension type</p>
+                  <select
+                    value={mode}
+                    onChange={e =>
+                      setMode(e.currentTarget.value as "block" | "widget" | "primitive")
+                    }>
+                    <option value="block">Block</option>
+                    <option value="widget">Widget</option>
+                  </select>
                 </div>
               </div>
               <div
                 style={{
+                  flex: 2,
                   display: "flex",
-                  justifyContent: "space-between",
+                  justifyContent: "center",
+                  alignItems: "center",
+                  padding: "0 40px",
                 }}>
-                <p>Extension type</p>
-                <select
-                  value={mode}
-                  onChange={e =>
-                    setMode(e.currentTarget.value as "block" | "widget" | "primitive")
-                  }>
-                  <option value="block">Block</option>
-                  <option value="widget">Widget</option>
-                </select>
+                <Icon icon="logo" />
```

Comment:
The logo isn't aligned vertically anymore.

Reply:
No correction will be made this time.